static/js/src/entry/business/SelectList.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div
+    :class="classes"
+  >
+    <label>
```

Comment:
This label should be tied to the `<select>` with a `for` attribute, referencing the `id` of the `<select>`.

Reply:
Thank you! Will address before production deploy.

Reply:
Question: That particular label is the "STATE" label above the field where the state selection shows. If I 'for' it it will show 'STATE' 51 times above the form field. Am I misunderstanding your intention with this comment?

Reply:
Different `for` -- not `v-for` but rather [this kind](https://stackoverflow.com/questions/698001/whats-the-for-for-in-a-label-tag). In other words, the `<label>` element should have an attribute `for="foo"`, and the `<select>` element should have `id="foo"`. That way, screen readers will be able to tie the text in the label to the actual form input.

Reply:
Ok. Got it. Thank you for the pointer to the example for clarification. 
Sincerely, 
Your humble Vue Nue-bie

Reply:
Vuebie!

Reply:
That works!